### PR TITLE
bump activities

### PR DESCRIPTION
**Pull request by spandanabaruah**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4364,7 +4364,7 @@
       }
     },
     "d2l-activities": {
-      "version": "github:BrightspaceHypermediaComponents/activities#4d200077220d94e6fcc5d352c1dc3ab40962c25a",
+      "version": "github:BrightspaceHypermediaComponents/activities#8f631f60baae112d1cd8f63673eb0ea9e693b0dc",
```
Comment (spandanabaruah) [inline package-lock.json:4367]: @wongvincent

Comment (wongvincent) [reply package-lock.json:4367]: Thanks!

**Comment by dlockhart** [comment]
@spandanabaruah is this still needed? Seems out of date by now.

**Comment by spandanabaruah** [comment]
> @spandanabaruah is this still needed? Seems out of date by now.

@dlockhart yup this is outdated now. Will close the PR.

**Pull request closed without merging**